Submit methods

evtimo opened the pull request:
By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Evgeny <zheny.tim@gmail.com>

evtimo committed:
Submit methods

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Evgeny <zheny.tim@gmail.com>

evtimo committed:
travis version fix

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Evgeny <zheny.tim@gmail.com>

akurilov commented on src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java:
```diff
@@ -21,132 +24,153 @@
+
+    @Override
+    protected int submit(List<O> ops) throws InterruptRunException, IllegalStateException {
+        for (final O op : ops) {
```
please use
```return submit(ops, 0, ops.size());```
instead of the loop

akurilov commented on .travis.yml:
```diff
@@ -21,23 +21,23 @@ before_install:
+    - docker login -u $DOCKER_USER -p $DOCKER_PASS
+    - export REPO=emcmongoose/mongoose-storage-driver-pravega
+    - export TAG=`if [ "$TRAVIS_BRANCH" == "master" ]; then echo "latest"; else echo $TRAVIS_BRANCH; fi`
+    - docker build --build-arg MONGOOSE_VERSION=4.0.0 -f docker/Dockerfile -t $REPO:$COMMIT .
```
please change the mongoose version to 4.0.1

akurilov commented on src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java:
```diff
@@ -21,132 +24,153 @@
+                        throw new AssertionError("Operation isn't supported");
+                }
+            } else {
+                throw new AssertionError("Operation type isn't supported");
```
"Operation type \\"{}\\" is not supported", op.getClass()

evtimo committed:
streamManager: Stream + Scope creation

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Evgeny <zheny.tim@gmail.com>

evtimo committed:
Revert "streamManager: Stream + Scope creation"

This reverts commit 2f95754d9b4e2dc451426b597d3759dac118b2bd.

akurilov commented on src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java:
```diff
@@ -21,132 +24,153 @@
+    @Override
+    protected int submit(List<O> ops, int from, int to) throws InterruptRunException, IllegalStateException {
+        for (int i = from; i < to; i++) {
+            submit(ops.get(i));
```
return value is ignored
please refer to the existing storage driver code (HDFS one for example)

evtimo committed:
fixing travis & submit + Maps

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

Signed-off-by: Evgeny <zheny.tim@gmail.com>
